crates/sui-indexer-alt-graphql/src/api/types/transaction_kind/programmable/commands/transaction_argument.rs

```diff
@@ -10,6 +11,7 @@ pub enum TransactionArgument {
     GasCoin(GasCoin),
     Input(Input),
     Result(TxResult),
+    Unknown(UnknownArgument),
```

Comment:
Translate this into a `None` instead of creating a new variant for it -- the `Unknown` variant is a gRPC thing -- we don't need to replicate it in GraphQL.